ouroboros-consensus-cardano/test/Test/ThreadNet/Cardano.hs

```diff
@@ -203,13 +235,62 @@ prop_simple_cardano_convergence TestSetup
     epochSizeShelley :: EpochSize
     epochSizeShelley = sgEpochLength genesisShelley
 
+    -- the protocol version of the Byron era proposal
+    propPV :: CC.Update.ProtocolVersion
+    propPV =
+      if setupHardFork
+      then
+        -- this new version must induce the hard fork if accepted
+        CC.Update.ProtocolVersion shelleyMajorVersion 0 0
+      else
+        -- this new version must not induce the hard fork if accepted
+        CC.Update.ProtocolVersion
+          byronMajorVersion (byronInitialMinorVersion + 1) 0
```

Comment:
Ah, so it is to test that proposals that should not trigger the transition to Shelley indeed don't, i.e., the goal of this PR.